### PR TITLE
filter some request and response headers to make diffs smaller

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: tests/vcr_python_wrapper.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 except ImportError:
     from urllib.parse import urlparse, urlunparse
 
+FILTER_REQUEST_HEADERS = ['Cookie']
+FILTER_RESPONSE_HEADERS = ['Apipie-Checksum', 'Date', 'ETag', 'Server', 'Set-Cookie', 'Via', 'X-Powered-By', 'X-Request-Id', 'X-Runtime']
```
Comment (mdellweg) [inline tests/vcr_python_wrapper.py:13]: what about removing everything starting with "X-"?

Comment (evgeni) [reply tests/vcr_python_wrapper.py:13]: I thought about that, but that would require a different filter approach (loop over existing headers and drop the ones with X-) so I didn't for now.

Comment (evgeni) [reply tests/vcr_python_wrapper.py:13]: Or did you mean add all the available X headers to that list?

File: tests/vcr_python_wrapper.py [hunk 2]
```diff
@@ -81,15 +84,23 @@ def host_body_matcher(r1, r2):
...
+        response['headers'].pop(header, None)
     return response
 
 
-def filter_request_uri(request):
+def filter_request(request):
+    # always use foreman.example.org as the name of the system
     request.uri = urlunparse(urlparse(request.uri)._replace(netloc="foreman.example.org"))
+
+    for header in FILTER_REQUEST_HEADERS:
```
Comment (evgeni) [inline tests/vcr_python_wrapper.py:99]: Hmm, technically those could go to `filter_headers` of VCR directly. That sounds cleaner.

Comment (evgeni) [reply tests/vcr_python_wrapper.py]: yeah, that's better.

File: tests/vcr_python_wrapper.py (hunk 1 above)
Comment (mdellweg) [reply tests/vcr_python_wrapper.py:13]: By nature of the X-headers, that would be impossible.
I thought about `if header in FILTER or header.startswith("X-"):`

Comment (evgeni) [reply tests/vcr_python_wrapper.py:13]: Yeah I meant all currently used ;)

Comment (evgeni) [reply tests/vcr_python_wrapper.py:13]: Anyways, let's merge this and think about more filtering later ;)

**Pull request merged**